src/CppUTest/CommandLineTestRunner.cpp

```diff
@@ -1,5 +1,6 @@
 /*
  * Copyright (c) 2007, Michael Feathers, James Grenning and Bas Vodde
+ * Copyright (c) 2019, Nokia
```

Comment:
We usually do not add the copyright of the company where the contributor works. If we would do so, there would be huge list of copyright notifications, not think that would benefit anyone.